R/stat-summary-bin.R

```diff
@@ -96,7 +96,11 @@ make_summary_fun <- function(fun.data, fun, fun.max, fun.min, fun.args) {
 
   if (!is.null(fun.data)) {
     # Function that takes complete data frame as input
-    fun.data <- match.fun(fun.data)
+    if (is.formula(fun.data)) {
+      fun.data <- rlang::as_function(fun.data)
+    } else {
+      fun.data <- match.fun(fun.data)
+    }
```

Comment:
I think you can just use `as_function()` for everything as it behaves correctly in all cases... We also import rlang so no need to prefix it